patches/gtimelog-appdata.patch

```diff
@@ -2,43 +2,12 @@ diff --git a/gtimelog.appdata.xml b/gtimelog.appdata.xml
 index 8bc4e75..78c3f99 100644
 --- a/gtimelog.appdata.xml
 +++ b/gtimelog.appdata.xml
-@@ -1,25 +1,30 @@
- <?xml version="1.0" encoding="UTF-8"?>
--<application>
--    <id type="desktop">gtimelog.desktop</id>
-+<component type="desktop-application">
-+    <id>gtimelog.desktop</id>
-     <metadata_license>CC0-1.0</metadata_license>
-     <project_license>GPL-2.0</project_license>
--
-     <name>Time Log</name>
-     <summary>Unobtrusively keep track of your time</summary>
-     <description>
-         <p>GTimeLog is a small GNOME app for keeping track of your time. Its main goal is to be as unobtrusive as possible.</p>
-     </description>
-     <screenshots>
--        <screenshot width="878" height="571" type="default">https://gtimelog.org/gtimelog.png</screenshot>
-+      <screenshot type="default">
-+        <image>https://gtimelog.org/gtimelog.png</image>
-+      </screenshot>
-     </screenshots>
-+    <developer_name>Time Log developers</developer_name>
-     <keywords>
-         <keyword>time</keyword>
-         <keyword>tracking</keyword>
-         <keyword>logging</keyword>
-     </keywords>
-+    <launchable type="desktop-id">gtimelog.desktop</launchable>
-     <url type="homepage">https://gtimelog.org/</url>
--    <updatecontact>marius@gedmin.as</updatecontact>
--    <component>
--        <translation type="gettext">gtimelog</translation>
--    </component>
--</application>
-+    <update_contact>marius@gedmin.as</update_contact>
-+    <translation type="gettext">gtimelog</translation>
-+    <content_rating type="oars-1.1" />
-+    <releases>
-+      <release version="0.11.3" date="2019-04-23" />
-+    </releases>
-+</component>
+@@ -25,7 +25,7 @@
+     <translation type="gettext">gtimelog</translation>
+     <content_rating type="oars-1.1" />
+     <releases>
+-      <release version="0.11.3" date="2019-04-23" />
++      <release version="0.12.0" date="2024-04-03" />
```

Comment:
in theory the new version would just be added.

better, this would be handled upstream as upstream do the releases.

Reply:
(I'm upstream.  I'll add a Makefile reminder to update this during the release process for next time.)

Should this list contain all past releases from https://pypi.org/project/gtimelog/#history, or is it meant only for Flatpak releases?

Reply:
it's not flatpak specific and GNOME software or equivalent use it too. And will show them.